common/js/src/popup-window/popup-opener.js

```diff
@@ -113,7 +141,7 @@ GSC.PopupOpener.runModalDialog = function(
     Object.assign(dataWithDialogCallbacks, opt_data);
 
   GSC.PopupOpener.createWindow(
-      url, createWindowOptions, dataWithDialogCallbacks);
+      url, DEFAULT_DIALOG_CREATE_WINDOW_OPTIONS, dataWithDialogCallbacks);
```

Comment:
Is this change intentional? We're calculating `createWindowOptions` above but are never using it.

If you mean that `opt_createWindowOptionsOverrides` is always unused now, then let's kill this argument and the `createWindowOptions` variable. Otherwise, we need to respect them.

Reply:
oh no this is actually an unintentional change

Reply:
oh no this is actually an unintentional change